pkg/scanner/scan_test.go

```diff
@@ -91,6 +104,18 @@ func TestScanner_ScanImage(t *testing.T) {
 						},
 					},
 				},
+				{
+					Target: "node-app/package-lock.json",
+					Vulnerabilities: []types.DetectedVulnerability{
+						{
+							VulnerabilityID:  "CVE-2019-11358",
+							PkgName:          "jquery",
+							InstalledVersion: "3.3.9",
+							FixedVersion:     ">=3.4.0",
+							LayerID:          "sha256:5216338b40a7b96416b8b9858974bbe4acc3096ee60acbc4dfb1ee02aecceb10",
+						},
+					},
+				},
```

Comment:
I'd like to know why this result doesn't contain `Type`.

Reply:
The mock was missing it, I added it.